### PR TITLE
Add chinese-lottery extension

### PR DESCRIPTION
**Pull request by elonwoo**
## Description

There are more than 300 million lottery players in China. 
This extension can help check the winning records of each lottery.

## Screencast

chinese-lottery-1

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/chinese-lottery/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Chinese Lottery ｜ 中国彩票
```
Comment (pernielsentikaer) [inline extensions/chinese-lottery/README.md:1]: Can we keep this README in english only?

File: extensions/chinese-lottery/package.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    "Data",
+    "News",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Chinese Lottery",
+      "subtitle": "福彩/体彩",
```
Comment (pernielsentikaer) [inline extensions/chinese-lottery/package.json:18]: Can we remove this

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/chinese-lottery/README.md (hunk 1 above)
Comment (elonwoo) [reply extensions/chinese-lottery/README.md]: ok, It's only English now.

File: extensions/chinese-lottery/package.json (hunk 2 above)
Comment (elonwoo) [reply extensions/chinese-lottery/package.json]: ok, i removed it.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**